Makefile

```diff
@@ -12,7 +12,10 @@ EXAMPLES = example/trilogy_query
 
 UNAME_S := $(shell uname -s)
 
-ifneq ($(UNAME_S), Darwin)
+ifeq ($(UNAME_S), Darwin)
+	CFLAGS += "-I$(shell brew --prefix openssl@1.1)/include"
+	LDFLAGS += "-L$(shell brew --prefix openssl@1.1)/lib"
+else
```

Comment:
In a perfect world this could be in its own pull, but I was getting sick of adding these myself locally when compiling so I'm personally all good with it going in with this.